plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/PluginUtil.kt

```diff
@@ -31,8 +32,11 @@ internal fun createConfiguration(target: Project, extension: KtlintExtension) =
         )
     }
 
-internal inline fun <reified T : Task> Project.taskHelper(name: String, noinline configuration: T.() -> Unit): T {
-    return this.tasks.create(name, T::class.java, configuration)
+internal inline fun <reified T : Task> Project.taskHelper(
```

Comment:
Can you call this `register`? Since the params are not the same, using `register` is still possible. It would be more intuitive than `taskHelper`. What do you think?